tasks/copy.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function (grunt) {
+  grunt.loadNpmTasks('grunt-contrib-copy');
+
+  grunt.config('copy', {
+    rdf: {
+      files: [
+        {
+          src: 'src/*.rdf',
```

Comment:
I'll still need to sort out a story for these pesky .rdf files. I have a few rough ideas in #44:

(a) [**lodash** RDF templates](https://github.com/mozilla/universal-search-addon/issues/44#issuecomment-120548503)
(b) [**jstoxml** and JSON schemas](https://github.com/mozilla/universal-search-addon/issues/44#issuecomment-121765108) that get converted to XML/RDF.

But the Grunt+**jstoxml** would work nicely with this Grunt flow.